tekton/tests/conftest.py

```diff
@@ -0,0 +1,121 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+import time
+from contextlib import ExitStack
+
+import pytest
+
+from datadog_checks.dev import get_here, run_command
+from datadog_checks.dev.kind import kind_run
+from datadog_checks.dev.kube_port_forward import port_forward
+
+HERE = get_here()
+
+
+def setup_tekton():
+    run_command(["kubectl", "create", "namespace", "tekton-operator"])
+    run_command(["kubectl", "apply", "-f", os.path.join(HERE, "kind", "tekton-operator.yaml"), "-n", "tekton-operator"])
+    time.sleep(30)
+    run_command(
+        ["kubectl", "wait", "pods", "--all", "--for=condition=Ready", "--timeout=300s", "-n", "tekton-operator"]
+    )
+    time.sleep(60)
```

Comment:
Curious, why all the sleeps here?

Reply:
I wanted to add a check on the endpoint but it's not available yet with our tooling though the container is ready :( so i just went with a sleep, which is not great but the env is consistently working here so I guess that's good enough.

Do you think I can improve that?

Reply:
Interesting, I don't have any off the top of my head right now, sadly. Maybe we can watch for something else. I'll circle back on this if I have some time, but this isn't a blocker.